packages/invoices/src/OutboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { memo } from 'react';
```

Comment:
This was interesting to layout. Possible a single large component is easier to read. Possibly having each of the sections in their own files is easier to read.

What do you think?

Reply:
I think the sections have mostly got to the size where it's better to split them. The side panel, especially for inbound, outbound & reqs, is only going to get bigger!